ext/rotoscope/rotoscope.h

```diff
@@ -12,35 +12,37 @@
 
 #define STACK_CAPACITY 500
 
-#define _RS_SHARED_CSV_HEADER "entity,method_name,method_level,filepath,lineno"
-#define _RS_SHARED_CSV_FORMAT "\"%s\",\"%s\",%s,\"%s\",%d"
-#define _RS_SHARED_CSV_VALUES(trace) \
-    StringValueCStr(trace.entity), \
-    StringValueCStr(trace.method_name), \
-    trace.method_level, \
-    StringValueCStr(trace.filepath), \
-    trace.lineno
-
-#define RS_CSV_HEADER "event," _RS_SHARED_CSV_HEADER
-#define RS_CSV_FORMAT "%s," _RS_SHARED_CSV_FORMAT
-#define RS_CSV_VALUES(trace) trace.event, _RS_SHARED_CSV_VALUES(trace)
-
-#define RS_FLATTENED_CSV_HEADER _RS_SHARED_CSV_HEADER ",caller_entity,caller_method_name,caller_method_level"
-#define RS_FLATTENED_CSV_FORMAT _RS_SHARED_CSV_FORMAT ",\"%s\",\"%s\",%s"
-#define RS_FLATTENED_CSV_VALUES(frame) \
-    _RS_SHARED_CSV_VALUES(frame.tp), \
-    StringValueCStr(frame.caller->tp.entity), \
-    StringValueCStr(frame.caller->tp.method_name), \
-    frame.caller->tp.method_level
+// clang-format off
+#define _RS_COMMON_CSV_HEADER "entity,method_name,method_level,filepath,lineno"
+#define _RS_COMMON_CSV_FORMAT "\"%s\",\"%s\",%s,\"%s\",%d"
+#define _RS_COMMON_CSV_VALUES(trace)  \
```

Comment:
For future reference, it is better to separate manual changes like this from automated changes from a script like clang-format to make the PR easier to review.